backend/WebApplication1/Services/PlateService.cs

```diff
@@ -1,77 +1,75 @@
 ﻿using RestSharp;
 using System.Collections.Generic;
 using System.Linq;
-using WebApplication1.Infrastructure.Utils;
 using WebApplication1.Models;
 using WebApplication1.Models.OpenALPR.Responses;
 using WebApplication1.Models.Responses;
 using WebApplication1.Repositories;
-using static WebApplication1.Models.Abstract.MissingModel;
 using WebApplication1.Infrastructure.Exceptions;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 using WebApplication1.Infrastructure.Timestampers.Abstract;
 using System.Threading.Tasks;
 using System;
+using WebApplication1.Infrastructure.Extensions;
 
 namespace WebApplication1.Services
 {
     public class PlateService
     {
         private readonly string _shKey = AppSettings.Configuration.AlprKey;
         private PlateRepository _plateRepository;
+        private TimestampRepository _timestampRepository;
         private readonly ITimestamper<Plate> _timestamper;
 
-        public PlateService(PlateRepository plateRepository, ITimestamper<Plate> timestamper)
+        public PlateService(PlateRepository plateRepository, TimestampRepository timestampRepository, ITimestamper<Plate> timestamper)
         {
             _plateRepository = plateRepository;
+            _timestampRepository = timestampRepository;
             _timestamper = timestamper;
```

Comment:
this is no good, we shouldn't need 2 objects for same-domain tasks, timestampRepository should handle both saving and retrieving timestamps

Reply:
nice catch.